### PR TITLE
added appveyor build script to properly build windows installer

### PR DESCRIPTION
**Pull request by nickalie**

### CODE REVIEW

File: installer.cfg [hunk 1]
```diff
@@ -18,6 +19,21 @@ packages = click
            colorama
            watchdog
            cookiecutter
+           zope
```
Comment (Moshe-Immerman) [inline installer.cfg]: I don't think we need all of these? especially twisted and zope?

Comment (nickalie) [reply installer.cfg]: "commons" package requires "twisted" and "zope". If you run `pip install commons`, these packages will be installed automatically. But while building windows installer these packages won't be included to the bundle and we need to explicitly specifies them in installer.cfg.

There're still issues with latest commit `utils.py`: method `print_fail` unavailable from the bundle, but it works fine if you install `pt` through `pip` or `python pt/pt.py`. I'm working on it, so please don't merge it yet

Comment (Moshe-Immerman) [reply installer.cfg]: "commons" isn't a dependency only "mcommons"

Comment (nickalie) [reply installer.cfg]: On windows using "mcommons" package throws
```
File "pt/pt.py", line 13, in <module>
    from client import Client
  File "D:\develop\projects\egis\papertrail-python-cli\pt\client.py", line 4, in <module>
    from commons import *
  File "d:\develop\tools\python\2.7\lib\site-packages\commons\__init__.py", line 14, in <module>
    from shell import *
  File "d:\develop\tools\python\2.7\lib\site-packages\commons\shell.py", line 3, in <module>
    import sh
  File "d:\develop\tools\python\2.7\lib\site-packages\sh.py", line 36, in <module>
```

Though "commons" works fine.

**Pull request merged**